effekt/shared/src/main/scala/effekt/Intelligence.scala

```diff
@@ -110,6 +119,16 @@ trait Intelligence {
     case u => C.definitionTreeOption(u)
   }
 
+  def getDocumentationOf(s: Symbol)(using C: Context): Option[Doc] =
+    getDefinitionOf(s).asInstanceOf[Option[Any]] match {
+      case Some(p: Product) =>
+        p.productElementNames.zip(p.productIterator)
+          .collectFirst {
+            case ("doc", Some(s: String)) => Some(s)
```

Comment:
I wonder if there's a way to get rid of the magic string literal here somehow :thinking:?

Reply:
Not sure, I am not aware of it. I also do not really have a problem with this implementation, I think it is precisely the solution which our way of annotating `Doc`s requires us to use.